### PR TITLE
Added VERY basic reactjs ui stuff

### PR DESCRIPTION
**Pull request by adundarov**
I added a pages folder in SRC with some pages and their respective css. I also modified some files like app.tsx and index.tsx to add navigation. I also updated package.json dependencies

### CODE REVIEW

**Comment by adundarov** [comment]
I see the issue, strangely didn't go off on my machine but the app.tsx changes seem to be not working well with the tests. Will fix shortly

**Review by jspaniac — APPROVED** [review]
Wow, nice! Lgtm!

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "equidistant",
```
Comment (zshafe) [inline package-lock.json]: I don't think that this needs to exist. The relevant package-lock.json is in the `my-app` directory, and this is empty anyway

File: frontend/my-app/src/Pages/Account.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
+.container {
+    display: flex;
```
Comment (zshafe) [inline frontend/my-app/src/Pages/Account.css:2]: While building out our own `.css` files and whatnot is great, I am curious if this was necessary, as yesterday we talked about using `material-ui`'s pre-styled components. Is that something you want to get into after the beta release?

File: frontend/my-app/src/Pages/Account.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                Upload Image
+            </button>
+            <button className="delete-account" onClick={handleDeleteAccount}>
+                Delete Account
+            </button>
+            <button className="back" onClick={handleBack}>
+                Back
+            </button>
+        </div>
+        <div className="right-column">
```
Comment (zshafe) [inline frontend/my-app/src/Pages/Account.tsx:50]: This feels like a lot of things that we could (and should) break down into multiple components, but for the purposes of getting a beta version out it looks good

**Review by zshafe — APPROVED** [review]
Overall looks good for a basic draft of UI stuff, but we'll want to do a lot of refactoring as soon as we have a MVP for the Beta version. Additionally, just curious about the use of basic HTML tags and straight CSS as opposed to using a component library. I also recommend (if you haven't seen it already) using the [new Beta React Documentation](https://beta.reactjs.org/) if you haven't seen them before. While technically still in Beta, they are 99% done at this point and have the most up to date style and tools. There's tons of outdated React things out there, so just wanted to be sure you knew about these

**Review by zshafe — CHANGES_REQUESTED** [review]
just drop the top level package-lock and i'm good w/ merging for now

**Review by zshafe — APPROVED** [review]
ship it

**Pull request merged**